minerva/backend/neptune.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
@jakubczakon are we going to introduce new file, or move this to `utils.py`

Reply:
sure - I can move it to utils.py